gearmand/datadog_checks/gearmand/data/conf.yaml.example

```diff
@@ -12,21 +23,31 @@ instances:
     #
     port: 4730
 
-    ## @param tasks - list of strings - optional
```

Comment:
how come the tasks parameter was removed?

Reply:
Forgot this one, nice catch